src/components/Auth/ForgotPasswordComponent.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
```

Comment:
Let's not use bindActionCreators and mapDispatchToProps. Just pass in an object containing all your actions as the second parameter in the `connect` function

Reply:
Alright